src/ReturnBookUI.java

```diff
@@ -1,87 +1,81 @@
 import java.util.Scanner;
 
-

```

Comment:
Removing empty line 'whitespace' is not established in the guidelines.
Discuss in meeting.

Reply:
This isn't an established guideline, it was done in an attempt to better read the code during the refactor.